sections/1-introduction.tex

```diff
@@ -3,28 +3,33 @@ \section{Introduction}
 
 Research astrophysicists rely on software to analyze increasingly large and complex data sets .
 Scientists that study our nearest star, the Sun, are no different.
-Solar physicists rely primarily on remote sensing data from both space- and ground-based observatories to measure the properties of our star and deduce the physical mechanisms at work.
+Solar physicists rely on remote sensing data from both space- and ground-based instruments to measure the properties of our star and deduce the physical mechanisms at work.
 In the past, most solar data analysis was performed using Fortran, a general-purpose, compiled programming language designed for scientific and engineering applications.
-In the 1980s, the solar physics community made the transition from Fortran to the Interactive Data Language (IDL), a commercial and closed-source programming language.
-The community extracted several benefits from this transition.
-IDL is a vector-oriented interpreted programming language which enables much faster and therefore cheaper development compared to Fortran.
-It also comes packaged with a large number of powerful libraries that support scientific data visualization and analysis.
-For similar reasons, the astronomy community also made the transition to IDL which led to the release of the IDL Astronomy User's Library for the solar community to leverage.
-Since then, significant and powerful functionality has been developed for solar data analysis as part of SolarSoftWare, a set of freely-available integrated IDL software libraries that provide a common data analysis environment \citep{Freeland:1998we}.
+Several reasons motivated the solar physics community to adopt the Interactive Data Language (IDL), a commercial and closed-source programming language, in the 1980s.
```

Comment:
Should there be a comma after language? Seems awkward, but I often forget rules about comma placement

Reply:
Yes, it is an independent clause.